packages/dcos-sdk/README.md

```diff
@@ -0,0 +1,52 @@
+# DC/OS UI SDK
+
+DC/OS plugin developer's best friend!
+
+## NavigationService
```

Comment:
Navigation service or extension provider?

Reply:
Service. A service wraps an extension provider, but technically not an extension provider on its own